lib/multicell_location/Kconfig

```diff
@@ -45,9 +45,10 @@ config MULTICELL_LOCATION_SERVICE_HERE
 config MULTICELL_LOCATION_SERVICE_SKYHOOK
 	bool "Skyhook location service"
 	select REST_CLIENT
+	depends on MODEM_JWT
 	help
 	  Use Skyhook location service. If a device ID is not provided for
-	  a location request, the device's IMEI will be used.
+	  a location request, the device's modem UUID will be used.
```

Comment:
It's no longer possible to provide a device ID, so I guess the second sentence should be removed. The same change should be done also for nRF Cloud service configuration.